src/pages/trade-types/cfds/index.tsx

```diff
@@ -1,18 +1,81 @@
-import React from 'react'
+import React, { ReactNode } from 'react'
 import Loadable from '@loadable/component'
 import { SmallContainer, Hero } from '../components/_style'
-import WhatIsCFD from './_what_is_cfd'
 import { SEO } from 'components/containers'
 import { Header } from 'components/elements'
 import Layout from 'components/layout/layout'
 import { localize, WithIntl } from 'components/localization'
+import TRADE_DUMMY from 'images/common/trade-type-dummy.png'
+import PLATFORM_DUMMY from 'images/common/platforms-dummy.png'
+import WCS_DUMMY from 'images/common/wcs-dummy.png'
+import CFD_DUMMY from 'images/common/cfd_dummy.png'
+import WHY_DUMMY from 'images/common/why-cfd-dummy.png'
+import START_DUMMY from 'images/common/start-cfd-dummy.png'
+import CFD_TRADE_DUMMY from 'images/common/cfd-trading-dummy.png'
+import CFD_MARKETS_DUMMY from 'images/common/cfd-markets-dummy.png'
+import { useHandleLazyLoad } from 'components/hooks/use-handle-lazy-load'
+const WhatIsCFD = Loadable(() => import('./_what_is_cfd'))
 const WhyTradeCFD = Loadable(() => import('./_why-trade-cfd'))
 const TradingCFDIncreases = Loadable(() => import('./_trading-cfd-increases'))
 const StartTrading = Loadable(() => import('./_start-trading'))
 const ThingsToKeep = Loadable(() => import('./_mind-when-trading'))
 const AvailableMarkets = Loadable(() => import('./_available-markets'))
 
 const CFD = () => {
+    const target = '#gatsby-focus-wrapper'
```

Comment:
It's better to move `target`, `options` and `lazy_components` and `lazyTemplate` outside the CFD!

Reply:
Yeah  sure i will keep it outside the CFD